rest_framework/compat.py

```diff
@@ -140,6 +140,16 @@ def value_from_object(field, obj):
     return field.value_from_object(obj)
 
 
+def getargspec(obj):  # type: tuple
+    if not hasattr(inspect, 'signature'):
```

Comment:
An inline comment referring to which python versions each branch applies to would make sense.